src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -323,38 +175,39 @@ type MessageDbContext(client: MessageDbClient, batchOptions: BatchOptions) =
     member val BatchOptions = batchOptions
 
     member _.TokenEmpty = Token.create -1L
```

Comment:
did I mention that I hate -1 ? maybe the store interface can use the term streamVersion (-1) based and the rest uses 0 based version/index?

Reply:
thinking I might move the conversion into the client and just deal with 0-based versions everywhere else. Making the token "data" `unit`

Reply:
yes, that was the objective of my whining, thanks!